### PR TITLE
KAZOO-3782

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

**Comment by fenollp** [comment]
@lazedo could you squash the escript changes into one commit? I like them and in conjonction to #1136 this should fix all our dialyzing troubles

**Comment by lazedo** [comment]
i prefer if you merge this PR.

File: applications/crossbar/src/cb_context.erl [hunk 1]
```diff
@@ -158,7 +159,7 @@ account_modb(Context, Year, Month) ->
     kazoo_modb:get_modb(account_id(Context), Year, Month).
 
 account_realm(Context) ->
-    wh_json:get_value(<<"pvt_realm">>, account_doc(Context)).
+    kz_account:realm(account_doc(Context)).
```
Comment (fenollp) [inline applications/crossbar/src/cb_context.erl]: `realm/1` fetches `<<"realm">>`, not `<<"pvt_realm">>`.

File: applications/crossbar/src/modules/cb_presence.erl [hunk 2]
```diff
@@ -69,42 +71,129 @@ resource_exists(_, ?RESET) -> 'true'.
 %%--------------------------------------------------------------------
 -spec validate(cb_context:context()) -> cb_context:context().
 -spec validate(cb_context:context(), path_token(), ne_binary()) -> cb_context:context().
-validate(#cb_context{req_verb = ?HTTP_GET, account_id=AccountId}=Context) ->
-    Doc = cb_context:doc(crossbar_doc:load(AccountId, Context)),
-    Realm = wh_json:get_value(<<"realm">>, Doc),
-    publish_search_req(Realm, Context).
+validate(#cb_context{req_verb = ?HTTP_GET}=Context) ->
+    publish_search(cb_context:req_param(Context, <<"type">>), cb_context:account_realm(Context), Context).
```
Comment (fenollp) [inline applications/crossbar/src/modules/cb_presence.erl]: …but if `account_realm/1` is only called from here I guess that's ok

File: applications/omnipresence/src/omnip_subscriptions.erl [hunk 3]
```diff
@@ -338,6 +338,15 @@ code_change(_OldVsn, State, _Extra) ->
 %%% Internal functions
 %%%===================================================================
 
+-spec notify_packages(atom(), subscriptions()) -> 'ok'.
+notify_packages(_MsgType, []) -> 'ok';
+notify_packages(MsgType , [#omnip_subscription{user=User
+                                           ,event=Package
+                                          }=Subscription | Subscriptions]) ->
```
Comment (fenollp) [inline applications/omnipresence/src/omnip_subscriptions.erl]: Indentation is broken

File: applications/crossbar/src/cb_context.erl (hunk 1 above)
Comment (lazedo) [inline applications/crossbar/src/cb_context.erl]: and it was not working :)

**Comment by fenollp** [comment]
Merging as soon as Travis agrees :)

**Pull request merged**